glib-sys/src/lib.rs

```diff
@@ -648,24 +651,8 @@ pub const G_DATE_BAD_YEAR: c_int = 0;
 pub const G_DIR_SEPARATOR: c_int = 92;
 pub const G_DIR_SEPARATOR_S: *const c_char = b"\\\0" as *const u8 as *const c_char;
```

Comment:
These two are platform specific (hello Windows)

Reply:
Yes, I just ignored it wrong.